download-chromedriver.js

```diff
@@ -1,41 +1,28 @@
-const { promises: fs } = require('fs')
+const fs = require('fs')
 const path = require('path')
-const { downloadArtifact } = require('@electron/get')
-const extractZip = require('extract-zip')
 const versionToDownload = require('./package').version
-
-function download (version) {
-  return downloadArtifact({
-    version,
-    artifactName: 'chromedriver',
-    platform: process.env.npm_config_platform,
-    arch: process.env.npm_config_arch,
-    rejectUnauthorized: process.env.npm_config_strict_ssl === 'true',
-    quiet: ['info', 'verbose', 'silly', 'http'].indexOf(process.env.npm_config_loglevel) === -1
-  })
-}
+const os = require('os');
+const { DownloaderHelper } = require('node-downloader-helper')
 
 async function attemptDownload (version) {
-  try {
-    const targetFolder = path.join(__dirname, 'bin')
-    const zipPath = await download(version)
-    await extractZip(zipPath, { dir: targetFolder })
-    const platform = process.env.npm_config_platform || process.platform
-    if (platform !== 'win32') {
-      await fs.chmod(path.join(targetFolder, 'chromedriver'), 0o755)
-    }
-  } catch (err) {
-    // attempt to fall back to semver minor
-    const parts = version.split('.')
-    const baseVersion = `${parts[0]}.${parts[1]}.0`
 
-    // don't recurse infinitely
-    if (baseVersion === version) {
-      throw err
-    } else {
-      await attemptDownload(baseVersion)
+    try {
+      const targetFolder = path.join(os.homedir(), 'Library/Caches/electron')
+      if (fs.existsSync(targetFolder + '/chromedriver-v10.1.5-darwin-x64.zip')) {
+        console.log('chromedriver already downloaded!')
+      } else {
+        const dl = new DownloaderHelper
+        ("https://github.com/electron/electron/releases/download/v10.1.5/chromedriver-v10.1.5-darwin-x64.zip", targetFolder)
+        dl.on('end', () => console.log('Download Completed'))
```

Comment:
`async attemptDownload` is an async function, you need to await promises or return a promise from here. otherwise, this will lead to a race condition where the file is still downloading but the script exits, or you do something else while the file is still downloading.

you need something like:
```js
await new Promise((resolve, reject) => {
  ...
  dl.on('end', resolve)
  dl.on('error', reject) // please handle errors!
});
```